server/src/main/java/org/elasticsearch/repositories/RepositoriesMetrics.java

```diff
@@ -36,7 +36,7 @@ public record RepositoriesMetrics(
     public static final String METRIC_UNSUCCESSFUL_OPERATIONS_TOTAL = "es.repositories.operations.unsuccessful.total";
     public static final String METRIC_EXCEPTIONS_HISTOGRAM = "es.repositories.exceptions.histogram";
     public static final String METRIC_THROTTLES_HISTOGRAM = "es.repositories.throttles.histogram";
-    public static final String HTTP_REQUEST_TIME_IN_MICROS_HISTOGRAM = "es.repositories.requests.http_request_time.histogram";
+    public static final String HTTP_REQUEST_TIME_IN_MILLIS_HISTOGRAM = "es.repositories.requests.http_request_time.histogram";
```

Comment:
I wonder if it is worth changing the metric name?
While the transition is in progress some time intervals will return metrics containing both micros and mills making the histogram tricky to interpret. 
Or may be we could use it as an opportunity to annotate corresponding metrics with release dates to avoid this confusion.

Reply:
> Or may be we could use it as an opportunity to annotate corresponding metrics with release dates to avoid this confusion.

This is on my todo list this iteration.

I think it's ok to reuse the same metric. In a week or two, we'll mostly see just the new data. I can add a note to the dashboard in the timebeing.

Reply:
Please keep in mind we might have a skewed results when querying for a longer time interval.
Anyway, I do not think this is a show stopped as old data is going to be eventually removed.